packages/legend-graph/src/graphManager/GraphManagerState.ts

```diff
@@ -172,6 +174,11 @@ export class GraphManagerState extends BasicGraphManagerState {
   dependenciesBuildState = ActionState.create();
   graphBuildState = ActionState.create();
   generationsBuildState = ActionState.create();
+  isPrimitiveTypeElement: (element: PackageableElement) => boolean;
```

Comment:
Let's not do this, you can have these methods directly declared on the class instead of having these as class attributes. see `isInstanceSetImplementation()` for example

Reply:
Oh makes sense, got it! 👍